### ISSUE TITLE
Automate Positive Add Roles CLI

### ISSUE THREAD

**Issue by renzon** [SatelliteQE/robottelo#3729]

**Comment by renzon** [comment]
[Stubbed test](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_user.py#L217). Tried play with add_role but was not able to figure out how it works:

``` console
[root@mgmt5 ~]# hammer -u admin -p ********* user add-role --id 5 --role-id 11,9Could not assign the user role:
  Error: option '--role-id': numeric value is required
  
  See: 'hammer user add-role --help'
[root@mgmt5 ~]# hammer -u admin -p ********* user add-role --id 5 --role-id 11 --role-id 9
User role has been assigned
[root@mgmt5 ~]# hammer -u admin -p ********* user info --id 5
Id:                   5
Login:                3edzhg
Name:                 9HnK1L09bh Ri0CsVpxxj
Email:                3edzhg@example.com
Admin:                yes
Authorized by:        Internal
Locale:               default
Timezone:             default
Last login:           2016/08/11 19:09:37
Default organization: 
Default location:     
Roles:                
    View hosts
    Anonymous
User groups:          

Created at:           2016/08/11 19:06:01
Updated at:           2016/08/11 19:09:37
```

Any doc or hint about how pass multiple roles at once?

**Comment by omaciel** [comment]
@renzon I believe `--role-id` takes a single value and you're passing multiple values to it.

**Comment by rochacbruno** [comment]
>  Any doc or hint about how pass multiple roles at once?

I guess the right way is the way you are already doing `add-role --id 5 --role-id 11 --role-id 9`

**Comment by renzon** [comment]
@rochacbruno I tried that but it just added the first role. It has two because "Anonymous" was alread there. Sorry for not better explain that before. I am going to fill a bug because if multiple values were not accepted I think console should present an error instead of discarding last role

**Comment by renzon** [comment]
Have created my first redmine issue because of this:  http://projects.theforeman.org/issues/16206

**Comment by renzon** [comment]
closed due to more general #3733
